tests/models/test_owlv2.py

```diff
@@ -57,3 +57,22 @@ def test_successful_image_detection_with_nms():
 
     for pred in results[0]:
         assert pred.label in prompts
+
+
+def test_successful_image_with_large_prompt():
+    test_image = "000000039769.jpg"
+    prompts = [
+        """a photo of a  cat that is spleeping next to a remote control,
+            the cat has a light brown color with black spots and seems to be wearing a light green necklace.
+            It also seems to be stretching its right leg and next to its left leg, it is stepping on the tail"""
+    ]
+
+    image = Image.open(f"tests/tools/data/owlv2/{test_image}")
+
+    owlv2 = Owlv2()
+
+    results = owlv2(prompts=prompts, image=image)
+    assert len(results[0]) > 0
+
+    for pred in results[0]:
+        assert pred.label == prompts[0]
```

Comment:
I don't fully understand this, should the label be the first work in prompts?

Reply:
So, remember that the output is a `list[list[BboxLabel]]`, and since we are sending just a single image, that is why we check for results[0], then the image could have multiple predictions `list[BboxLabel]` and we do a for loop over that. The reason of `label == prompts[0]` is because we can have multiple prompts searching for multiple objects but in this case we just have a single prompt, the long prompt. Finally, the assigned label to the predicted `BboxLabel` is the same as each of the given `prompt` values (which in this case is only one).